server/routerlicious/packages/lambdas/src/scribe/lambda.ts

```diff
@@ -438,12 +438,24 @@ export class ScribeLambda implements IPartitionLambda {
         }
     }
 
+    /**
+     * Updates protocol head to the new summary sequence number
+     * Protocol head is the sequence number of the last summary
```

Comment:
nit: Reversing the order of the first 2 lines would probably make it clearer. I got a little confused because the second line and the param comment are contradictory and it took me a couple of reads to get it in :-).